molgenis-data/src/main/resources/l10n/data_en.properties

```diff
@@ -0,0 +1,11 @@
+D01=Unknown entity type ''{0}''
+D02=Unknown entity ''{0}'' of type ''{1,label}''
+D04=Unknown attribute ''{1}'' of entity type ''{0,label}''
+D05=Unknown repository ''{0}''
+D06=Unknown repository collection ''{0}''
+D07=Unknown package ''{0}''
+D08=Unknown parent entity type''{0}''
+D10=Entity type ''{0,label}'' attribute ''{1,label}'' value ''{2}'' can''t be converted to type ''{3}''
```

Comment:
add missing dots

Reply:
proposal: do nothing, updating messages is out of scope for the global exception handling feature.